slimta/logging/__init__.py

```diff
@@ -128,7 +130,7 @@ def logline(log, type, typeid, operation, **data):
         log('{0}:{1}:{2}'.format(type, typeid, operation))
     else:
         data_str = ' '.join(['='.join((key, log_repr.repr(val)))
-                             for key, val in sorted(data.iteritems())])
+                             for key, val in sorted(six.iteritems(data))])
```

Comment:
That was my thought at first too, but a lot of these cases are very bounded, e.g. there should never be enough keys in this dict for it to be significant. I think that's true of many of the other cases of this as well.